### PR TITLE
EJDOTNETCORE-4344-1 Added API link and removed unwanted code space in features md file and updated the issue changes

### PR DESCRIPTION
**Pull request by sameerkhan001**
Hi Sir,

Added API link and removed unwanted code space in features md file and updated issue changes

Regards,
Sameerkhan N

### CODE REVIEW

File: File-Formats/PDF/Working-with-Pages.md [hunk 1]
```diff
@@ -1677,7 +1677,7 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Rotating an existing PDF page
 
-You can also rotate a PDF page in the existing PDF document using PdfPageRotateAngle(https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.PdfPageRotateAngle.html) as shown in the following code snippet.
+You can also rotate a PDF page in the existing PDF document using [PdfPageRotateAngle](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.PdfPageRotateAngle.html) as shown in the following code snippet.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Working-with-Pages.md:1680]: change as like 
 following code example

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 2]
```diff
@@ -10,20 +10,17 @@ documentation: UG
 
 ## URL to PDF
 
-To convert website URL or local HTML file to PDF document, refer to the following code snippet.
+To convert website URL or local HTML file to PDF document using [Convert](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.HtmlToPdfConverter.html#Syncfusion_HtmlConverter_HtmlToPdfConverter_Convert_System_String_) method, refer to the following code snippet.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:13]: using [Convert]() method in [HtmlToPdfConverter]() class.

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 3]
```diff
@@ -268,7 +241,7 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## JavaScript
 
-The Blink HTML converter supports enabling or disabling the JavaScript while converting HTML to PDF. Refer to the following code snippet.
+The Blink HTML converter supports enabling or disabling the JavaScript using [EnableJavaScript](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.BlinkConverterSettings.html#Syncfusion_HtmlConverter_BlinkConverterSettings_EnableJavaScript) property while converting HTML to PDF. Refer to the following code snippet.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:244]: [EnableJavaScript]() property in BlinkConverterSettings class

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 4]
```diff
@@ -382,7 +340,7 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Hyperlinks
 
-The Blink HTML converter support preserving URL links from HTML to PDF. Refer to the following code snippet.
+The Blink HTML converter support preserving URL links from HTML to PDF using [EnableHyperLink ](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.BlinkConverterSettings.html#Syncfusion_HtmlConverter_BlinkConverterSettings_EnableHyperLink) property. Refer to the following code snippet.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:343]: using [EnableHyperLink]  - Remove the space 
using [EnableHyperLink]() property in [BlinkConverterSettings]() class

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 5]
```diff
@@ -929,7 +788,7 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Windows authentication
 
-The webpage you want to convert may protected with windows authentication. Blink rendering engine provides support for converting the Windows Authenticated webpage to PDF document by providing the username and password. Refer to the following code snippet.
+The webpage you want to convert may protected with windows authentication. Blink rendering engine provides support for converting the Windows Authenticated webpage to PDF document by providing the username and password using [BlinkConverterSettings](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.BlinkConverterSettings.html) class. Refer to the following code snippet.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:791]: Add link for username and password

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 6]
```diff
@@ -988,7 +837,7 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Form authentication
 
-The Blink HTML converter provides support for form authentication by using cookies. The cookies are send to web server for form authentication when the HTML page is requested. Each cookie is represented by a name and value. Refer to the following code snippet.
+The Blink HTML converter provides support for form authentication by [CookieCollection](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.CookieCollection.html) class using [Cookies](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.BlinkConverterSettings.html#Syncfusion_HtmlConverter_BlinkConverterSettings_Cookies) property. The cookies are send to web server for form authentication when the HTML page is requested. Each cookie is represented by a name and value. Refer to the following code snippet.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:840]: The Blink HTML converter provides support for form authentication by using [Cookies]() in [BlinkConverterSettings]() class

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 7]
```diff
@@ -1052,32 +888,24 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Token-based authentication
 
-The Blink HTML converter supports token-based authentication by using the HTTP request headers. The token values will be send to web server when the HTML page is requested. Refer to the following code snippet.
+The Blink HTML converter supports token-based authentication by [HttpRequestHeaderCollection](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.HttpRequestHeaderCollection.html) class using the [HttpRequestHeaders](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.BlinkConverterSettings.html#Syncfusion_HtmlConverter_BlinkConverterSettings_HttpRequestHeaders) property. The token values will be send to web server when the HTML page is requested. Refer to the following code snippet.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:891]: The Blink HTML converter supports token-based authentication by using the [HttpRequestHeaders]() property in [BlinkConverterSettings]() class.

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 8]
```diff
@@ -1116,7 +937,7 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Offline conversion
 
-The Blink HTML converter supports converting HTML to PDF in offline mode. While converting HTML to PDF in offline mode, the converter does not access the resources from the internet. This may increase the performance in slow internet connection.
+The Blink HTML converter supports converting HTML to PDF in offline mode using [EnableOfflineMode](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.BlinkConverterSettings.html#Syncfusion_HtmlConverter_BlinkConverterSettings_EnableOfflineMode) property. While converting HTML to PDF in offline mode, the converter does not access the resources from the internet. This may increase the performance in slow internet connection.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:940]: [EnableOfflineMode]() property in [BlinkConverterSettings]() class

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md [hunk 9]
```diff
@@ -1505,7 +1261,7 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## HTML to single PDF page
 
-By using this option, you can render the whole HTML content into a single PDF page. The PDF page size is limited to 14400 points. There are two options to enable this feature since this is disabled by default.
+By using this [SinglePageLayout](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.HtmlToPdf.SinglePageLayout.html) Enum, you can render the whole HTML content into a single PDF page. The PDF page size is limited to 14400 points. There are two options to enable this feature since this is disabled by default.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/features.md:1264]: use the SinglePageLayout property here, not enum

File: File-Formats/PDF/Working-with-Pages.md (hunk 1 above)
Comment (sameerkhan001) [reply File-Formats/PDF/Working-with-Pages.md:1680]: Changed

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md (hunk 2 above)
Comment (sameerkhan001) [reply File-Formats/PDF/Convert-HTML-To-PDF/features.md:13]: Modified

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md (hunk 3 above)
Comment (sameerkhan001) [reply File-Formats/PDF/Convert-HTML-To-PDF/features.md:244]: Modified

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md (hunk 4 above)
Comment (sameerkhan001) [reply File-Formats/PDF/Convert-HTML-To-PDF/features.md:343]: Modified

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md (hunk 5 above)
Comment (sameerkhan001) [reply File-Formats/PDF/Convert-HTML-To-PDF/features.md:791]: Modified

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md (hunk 6 above)
Comment (sameerkhan001) [reply File-Formats/PDF/Convert-HTML-To-PDF/features.md:840]: Modified

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md (hunk 7 above)
Comment (sameerkhan001) [reply File-Formats/PDF/Convert-HTML-To-PDF/features.md:891]: Modified

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md (hunk 8 above)
Comment (sameerkhan001) [reply File-Formats/PDF/Convert-HTML-To-PDF/features.md:940]: Modified

File: File-Formats/PDF/Convert-HTML-To-PDF/features.md (hunk 9 above)
Comment (sameerkhan001) [reply File-Formats/PDF/Convert-HTML-To-PDF/features.md]: modified

**Review by praveenHari24 — APPROVED** [review]

**Pull request merged**